recipes/GKE Setup/README.md

```diff
@@ -0,0 +1,105 @@
+# Google Kubernetes Engine (GKE) Setup
+
+There are many ways to set up hauser to run in a cloud environment; this guide does not
+aim to be comprehensive, but rather to provide a recommended configuration for a simple workflow.
+
+In order to be able to set up the infrastructure for your project, it's best if you have at an [Editor
+role](https://cloud.google.com/iam/docs/understanding-roles#basic). Else, you will need permission to
+create/edit the following resources:
+* Google Cloud Storage (GCS) buckets
+* BigQuery Datasets
+* GKE Kubernetes objects
+* Service Accounts
+* Service Account IAM Policies
+
+Ensure that you have the [Google Cloud SDK](https://cloud.google.com/sdk/docs/install) and `kubectl` installed
+and that you've [enabled cluster access](https://cloud.google.com/kubernetes-engine/docs/how-to/cluster-access-for-kubectl).
+
+To run the `./deploy.sh` script, you must also have the following CLI tools installed and in your $PATH
+* `gsutil` - Storage CLI (installable with `gcloud components install gsutil`)
+* `bq` - BigQuery CLI (installable with `gcloud components install bq`)
+* [`jq`](https://stedolan.github.io/jq/) - JSON processor
+
+## Prerequisites
+As described in the [cluster access documentation](https://cloud.google.com/kubernetes-engine/docs/how-to/cluster-access-for-kubectl),
+first, ensure that your gcloud SDK is initialized:
+```shell
+gcloud init
+```
+This will authenticate the SDK, allow you to select a project, and configure a default zone/region.
+
+Then (assuming you are using an existing cluster) you must generate a kubernetes configuration file:
+```shell
+gcloud container clusters get-credentials "<cluster-id>"
+```
+Before running the `deploy.sh` script, you must
+[create a BigQuery dataset](https://cloud.google.com/bigquery/docs/datasets#create-dataset) (or use an existing dataset)
+and a [Google Storage bucket](https://cloud.google.com/storage/docs/creating-buckets).
+
+As a default, the script expects the name of the dataset to be `fullstory-export-data` and the
+name of the bucket to be `$GOOGLE_PROJECT-hauser`, but feel free to change these.
+
+If you did not use the default dataset and bucket names, or want to customize the name of the service account,
+feel free to edit the following variables at the top of the `deploy.sh` script.
+```shell
+BIGQUERY_DATASET="fullstory-export-data"
+SERVICE_NAME="hauser-svc"
+GCS_BUCKET="$GOOGLE_PROJECT-hauser"
+HAUSER_VERSION="latest"
+```
+To find your FullStory Org Id, see [help docs](https://help.fullstory.com/hc/en-us/articles/360047075853-How-do-I-find-my-FullStory-Org-Id-).
+You will also need to create or get a [FullStory API Key](https://help.fullstory.com/hc/en-us/articles/360020624834-Where-can-I-find-my-API-key-).
+
+The `HAUSER_VERSION` variable corresponds to the docker image tag. See [docker hub](https://hub.docker.com/r/fullstorydev/hauser) for available tags.
```

Comment:
Maybe mention that the latest available version will be used by default?

Reply:
Also, might move this up above to line 40-ish. That way you introduce all of the parameters and their defaults first, and then the following paragraph describes how to change them by editing values in the script.